Snail/Observable.swift

```diff
@@ -298,7 +298,10 @@ public class Observable<T>: ObservableType {
         let combined = Observable<(T, U)>()
 
         var value1: T?
+        var input1Completed = false
```

Comment:
is there a better data structure to use here?

Reply:
I was thinking of using `combineLatest` here but then realized we are in the combineLatest function 🤦🏼 . We could use `nil` but that has the same effect as `Bool`

Reply:
Can use a boolean array as well but might be overkill here

Reply:
or a tuple for the (T?, Bool) - so the values are at least associated.  or a struct to wrap unless its overkill?

Reply:
I like the tuple, let me update